.travis/update-version.sh

```diff
@@ -1,47 +1,50 @@
 #!/usr/bin/env bash
 
-cd `dirname $0`/..
-
 setup_git() {
-  git config --global user.email "travis@travis-ci.org"
-  git config --global user.name "Travis CI"
+    echo "Setting up git config"
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "Travis CI"
 }
 
 update_version() {
-    # Push the new tag with `-SNAPSHOT` as the current version
+    echo "Updating the version in the POMs"
+
+    # Push the tag with `-SNAPSHOT` as the current version
     mvn --settings .travis/settings.xml org.codehaus.mojo:versions-maven-plugin:2.7:set -DnewVersion="${TRAVIS_TAG}-SNAPSHOT"
 
     # Increment the patch version
     mvn --settings .travis/settings.xml release:update-versions -B
 
-    # Pull the value from the pom
+    # Pull the new value from the pom
     NEW_VERSION=$(mvn --settings .travis/settings.xml help:evaluate -Dexpression=project.version -q -DforceStdout)
 }
 
 commit_files() {
+    echo "Commiting the changes to git"
 
-  # Use the version as the branch name
-  git checkout -b ${NEW_VERSION}
+    # Use the version as the branch name
+    git checkout -b ${NEW_VERSION}
```

Comment:
do we want to print git status afterwards?

Reply:
Done